src/js/tools/utils.ts

```diff
@@ -74,3 +75,22 @@ export function determineDebugIdFromBundleSource(code: string): string | undefin
   );
   return match ? match[1] : undefined;
 }
+
+/**
+ * CountingSet was added in Metro 0.71.0 before that NodeJS Set was used.
+ */
+function resolveSetCreator(): () => CountingSet<string> {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { default: MetroSet } = require('metro/src/lib/CountingSet');
+    return () => new MetroSet();
```

Comment:
l: should we debug log?

Reply:
I was thinking about it, but the RN CLI doesn't forward the output.